crates/edr_napi/Cargo.toml

```diff
@@ -17,7 +17,7 @@ log = { version = "0.4.20", default-features = false }
 napi = { version = "2.16.0", default-features = false, features = ["async", "error_anyhow", "napi8", "serde-json"] }
 napi-derive = "2.16.0"
 edr_defaults = { version = "0.3.5", path = "../edr_defaults" }
```

Comment:
Is there a way to link these versions to the workspace too?

Reply:
Yep there is a follow up task: https://github.com/NomicFoundation/edr/issues/467